components/dashboard/src/index.css

```diff
@@ -35,23 +35,29 @@
 
 @layer components {
     button {
-        @apply cursor-pointer px-4 py-2 my-auto bg-green-600 hover:bg-green-700 border-2 border-green-800 text-gray-100 text-sm font-medium rounded-md focus:outline-none focus:ring focus:border-blue-300;
```

Comment:
**issue**: We're going to need the border-2 for secondary buttons, see delete button in Account. ❗